app/services/build_json.rb

```diff
@@ -0,0 +1,21 @@
+class BuildJSON
+
+  def self.call(event, question)
+    choices_array = []
+
```

Comment:
I do think this can be refactored into inject. Will try after it is merged. What's up with all the blank lines?